get_started/what_is_citus.rst

```diff
@@ -35,7 +33,7 @@ Citus scales horizontally by adding worker nodes, vertically by upgrading worker
     * 2.4TB memory, 1280 cores, and 60TB of data
     * ...with plans to grow to 45 nodes
 * `Mixrank <https://www.citusdata.com/customers/mixrank>`_
-    * 160TB of time series data
+    * 1600TB of time series data
```

Comment:
Minor, but 1.6PB sounds better.

Reply:
(This has been corrected but github still shows the comment perhaps because it's attached to an individual commit)